src/Message/ApplePayAuthorizeRequest.php

```diff
@@ -411,10 +413,14 @@ public function sendData($data)
 
         /** @var EventDispatcherInterface|null $eventDispatcher */
         $eventDispatcher = $this->httpClient->getEventDispatcher();
+        $action = 'startSession';
```

Comment:
It looks like `$data` differs from the Vindicia AbstractRequest in that it doesn't have an 'action' key so I'm not able to tell which actions this request currently supports. It seems to be just for initiating a session atm? Once we add capture and authorize we can refactor this but for now I've hard-coded the only action that this request does.

Reply:
it's just supposed to log the request name, right? how about just give it a nice request name instead of calling something an "action"? action is a vindicia api specific thing